processing/src/main/java/io/druid/segment/incremental/OnheapIncrementalIndex.java

```diff
@@ -257,6 +257,15 @@ private void doAggregate(
     rowContainer.set(null);
   }
 
+  private void closeAggregators(Map<Integer, Aggregator[]> intMap)
+  {
+    for (Aggregator[] aggregators : intMap.values()) {
```

Comment:
@freakyzoidberg huh, I didn't realise it doesn't. This use-case is a good reason to make to extend `Closeable`.